OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -401,6 +401,14 @@ void trackBulkDownloadSwitchOn(@NonNull String courseId, int totalDownloadableVi
      */
     void trackDownloadToSdCardSwitchOff();
 
+    /**
+     * Track the fetched parameters from the Firebase Remote Configs server.
```

Comment:
Track the parameters relevant to the experiment of Firebase Remote Configs.
Ref: https://openedx.atlassian.net/browse/LEARNER-7394